### ISSUE TITLE
page request: qmake, autoconf, automake, libtool

### ISSUE THREAD

**Issue by pixelcmtd** [tldr-pages/tldr#6320]
- [x] [qmake](https://doc.qt.io/qt-5/qmake-manual.html) → #15507
- [x] [autoconf](https://www.gnu.org/software/autoconf/) → #8508
- [x] [automake](https://www.gnu.org/software/automake/) → #18785
- [x] [libtool](https://www.gnu.org/software/libtool/) → #11454

**Comment by CleanMachine1** [comment]
Not to be rude, but you may know more about this. 

I'm sure you would be able to do a better job.

**Comment by chrissxYT** [comment]
> you may know more about this.
> I'm sure you would be able to do a better job.

Not at all, I do not like it and only tried using it once. I think it's among `cmake` and `autoconf`/`automake` as bad ideas.
